bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/link/ItemChannelLinkRegistry.java

```diff
@@ -90,31 +87,6 @@ public ItemChannelLink update(ItemChannelLink element) {
         return linkedItems;
     }
 
-    /**
-     * Returns a set of bound things for the given item name.
-     *
-     * @deprecated Will be removed soon. Use {@link ItemThingLinkRegistry#getLinkedThings(String)} instead.
-     *
-     * @param itemName
-     *            item name
-     * @return set of bound things for the given item name
-     */
-    @Deprecated
```

Comment:
> Discussed with Kai already. I think, this deprecation is an error.

We still support item thing links?

> This method does nothing have to do with the ItemChannelLinkRegistry

If it is the wrong place and we support item thing links, then the function still needs to be removed from ItemChannelLinkRegistry... and you need to create an ItemChannelLink... or whatever.